common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -1572,12 +1537,6 @@ func (d *cassandraPersistence) ConflictResolveWorkflowExecution(request *p.Inter
 			State:           state,
 			Status:          status,
 		})
-
-		replicationVersions, err := serialization.ReplicationVersionsToBlob(
```

Comment:
`serialization.ReplicationVersionsToBlob` also should be removed. How can it builds if you removed `persistencespb.ReplicationVersions` ?

Reply:
oh.. you forgot to run `make proto`.